src/app/Main/Body/QueryPage/Results/TableResults/TableContent-TableContainer.component.tsx

```diff
@@ -43,11 +44,13 @@ const useStyles = makeStyles({
  * @returns {JSX.Element} - table container component
  */
 export function TableContentTableContainerComponent(props: {
-    sumOptions: { showRowSum: boolean; showColSum: boolean };
-    isCol: boolean;
+    sumOptions: SumOptions;
     tableData: Record<string, string>[];
     displayOption: DisplayOptionType;
+    displayRowCol: DisplayRowCol;

```

Comment:
I do not understand what this means. Can you please explain (perhaps again) what the difference is if isRow or isCol is true or false? Has this something todo if the user already selected what to display or not?

Reply:
The frontend has two feature selectors. One for the attribute in the rows and one for the columns. The display works when both are selected, but also when only one is selected. So the question is "isCol? True : False" and "isRow? True : False".

Reply:
Thanks, i remember now, if only one axis is selected only the sum of the selected isolates is shown.